scripts/fetch_tools.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+ARCH="$(uname -p)"
+DEST_DIR="${DEST_DIR:-_output/bin}"
```

Comment:
This implies the script can only be run from the top-level microshift directory. Let's make is relative to the script location?